test/functional/http/get/accounts.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
```

Comment:
Please restructure the tests using the pattern from the other functional tests introduced by @diego-G :
```
describe('GET /api/accounts')
   describe('?')
      describe('address')
      describe('publicKey')
      ....

```